README.md

```diff
@@ -64,6 +64,18 @@ to always keep macOS on and available.
 | `node['macos']['network_time_server']`  | `'time.windows.com'`    |
 | `node['macos']['time_zone']`            | `'America/Los_Angeles'` |
 
+**N.b.** When ChefSpec testing implementations of this recipe, the `node['hardware']['machine_model']`
```

Comment:
Quid est? Could this just be **'Note'**?

Reply:
It's latin for "nota bene" meaning "note well"